### PR TITLE
refactor: use new bucket for dtd-catalog

### PR DESCRIPTION
**Pull request by micdes-pagopa**
Apply when frontend/job ready

### CODE REVIEW

**Comment by micdes-pagopa** [comment]
cc: @Carminepo2

File: kubernetes/configs/dev/dtd_catalog_exporter.sh [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-DTD_CATALOG_STORAGE_BUCKET="interop-dtd-share-dev"
+DTD_CATALOG_STORAGE_BUCKET="interop-public-catalog-dev"
```
Comment (galales) [inline kubernetes/configs/dev/dtd_catalog_exporter.sh:3]: is this a public bucket?

Comment (micdes-pagopa) [reply kubernetes/configs/dev/dtd_catalog_exporter.sh:3]: Nope, private bucket with the landing page CDN serving its content

Comment (galales) [reply kubernetes/configs/dev/dtd_catalog_exporter.sh:3]: 🔝

**Review by galales — DISMISSED** [review]

**Comment by micdes-pagopa** [comment]
Increased cronjob frequency to once every 3 hours

**Review by galales — APPROVED** [review]

**Pull request merged**